File: Chapter_01_Introduction/Getting_Started_The_Easy_Problems/Super_Easy/12372-Packing_for_Holiday/main.cpp
/*
 * UVa 12372 - Packing for Holiday
 * Just check if all L, W, H <= 20
 */
#include <iostream>
#include <sstream>

int main()
{
  const int SIZE {20};
  std::ios_base::sync_with_stdio(false);
  std::cin.tie(nullptr);

  std::ostringstream output;
  int testCases {};
  std::cin >> testCases;
  int test {1};
  while (testCases--) {
    int length {}, width {}, height {};
    std::cin >> length >> width >> height;
    output << "Case " << test++ << ": "
      << (length <= SIZE && width <= SIZE && height <= SIZE
          ? "good" : "bad") << '\n';
  }
  std::cout << output.str();
}

File: Chapter_01_Introduction/Getting_Started_The_Easy_Problems/Super_Easy/12372-Packing_for_Holiday/Main.java
/*
 * Problem 12372 - Packing for Holiday
 * Java version 6/10/2016
 */

import java.io.PrintStream;
import java.util.Scanner;

public class Main {
    static final Scanner input = new Scanner(System.in);
    static final PrintStream output = System.out;

    public static void main(String[] args) {

        StringBuffer results = new StringBuffer();
        int testCases = input.nextInt();
        for (int i = 0; i < testCases; ++i) {
            int length = input.nextInt();
            int width = input.nextInt();
            int height = input.nextInt();

            results.append("Case " + (i + 1) + ": ");
            if (length <= 20 && width <= 20 && height <= 20) {
                results.append("good\n");
            } else {
                results.append("bad\n");
            }
        }
        System.out.print(results.toString());
    }
}
